docs/reference/api/drivers/AnalogOut.md

```diff
@@ -1,6 +1,6 @@
 ## AnalogOut
 
-Use the AnalogOut interface to set the voltage of an analog output pin as a floating point number from 0.0 (VSS) to 1.0 (VCC).
+Use the AnalogOut interface to set the voltage of an analog output pin as a floating point number from 0.0 (VSS) to 1.0 (VCC). DAC Resolution depends on the hardware device used, the max resolution supported is VCC/65,536V.
 
```

Comment:
@studavekar - "Use the AnalogOut interface to set the output voltage of an analog output pin specified as a percentage or as an unsigned short in the range 0x0000-0xFFFF. Separate API are provided to use percentage or range. Mbed-OS supports a maximum resolution of 1/65535 of the maximum voltage, although actual resolution ,,,, "